### DISCUSSION TITLE
I am not impressed with SDXL.

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1926] [General]
Maybe I'm just disappointed as an early adopter or something, but I'm not impressed with the images that I (and others) have generated with SDXL.

Maybe it's going to get better as it matures and there are more checkpoints / LoRAs developed for it.

But for photorealism, SDXL in it's current form is churning out fake looking garbage.

Has anyone here had a different experience ? 

If so, please point me in the right direction :)

Thanks !

**Comment by jlitz** [comment]
Try the dreamshaper model  helps   https://civitai.com/models/112902

**Comment by Verszy** [comment]
It's a base model. If you compare XL to 1.x or 2.x, it looks pretty crazy if you set aside things like VRAM usage for now. The style range it has is pretty baffling to me after all this time trying to achieve the same from finetuning what we already had. Compared to a 1.x or 2.x finetune or merge, well it's not going to be pulling you away from those. The thing to look forward to most is seeing what people train into that model and whether we can deliver powerful nudges without paving over as much of the base model's capabilities thanks to the larger parameter count.
